### ISSUE TITLE
The Yathsou (T3 Submarine Hunter) has become too strong after its recent buff

### ISSUE THREAD

**Issue by Basilisk3** [FAForever/fa#5894]
The 28 % DPS buff from #5834 in combination with its very effective torpedo defense, makes creating death blobs of this unit trivial. At the moment, this unit's performance is roughly comparable to how Sniper Bots were before their various nerfs. 

|   | Yathsou | Barracuda | Vesper |
| ------------- | ------------- | ------------- | ------------- |
| HP per mass | 1.333 | 1.3 | 1.636 |
| DPS per mass | 0.121 | 0.102 | 0.082 |
| Range | 65 | 45 | 45 |
| Anti-Air | Yes | No | No |

It has more DPS/mass than both Tech 2 Sub Hunters, while also having anti-air capabilities and 20 additional range.

**Comment by Basilisk3** [comment]
I apologize for this wall of text, but as there have been some questions as to why this is necessary, I will provide some more context:

**There have been three direct/indirect buffs to this unit**
- #5000 Essentially reworked how torpedoes work entirely. One of the most important changes was, that torpedo defenses now work against the torpedoes/depth charges of torpedo bombers. Additionally, they also work better in general. These were good changes, but also an immense buff to the Yathsou. In combination with its excellent torpedo defense (one of the best in the game) these changes result in almost unkillable death blobs.

- The next noteworthy change was in #5834, which was around a 28% DPS buff.

- On top of that, there was another indirect buff, one that makes torpedoes retarget after their original target has been destroyed. This is very import for this unit because of its high damage, and high range.

I believe all of the above changes were a positive addition to the game, but in combination they result in this unit becoming too powerful.

**Comment by zqh9843** [comment]
Oh so torpedoes can actually retarget just like the sam missile now? Lemme do a test real quick

**Comment by zqh9843** [comment]
Here are the results of my tests summarized:
I can confirm that in a equivalent mass fight where both doomstacks come at each other sera will usually win with about 0%-15% units surviving, the bigger the doomstack the more sera subs survive in the end. Although, when in low number, fights are very close with Aeon T2 sub enjoying more successes compared with cybran sub hunters.
However imo those doomstack scenarios aren't very likely since both sides are investing like at least 30000 mass into the sub production and a team with air can easily invest that much into torp bomber productions.
Can u pack up the changes and upload it as a mod so more players can test?

**Comment by Garanas** [comment]
@zqh9843 you can run the code by setting up a development environment. You can learn more about that [here](https://github.com/FAForever/fa/blob/deploy/fafdevelop/setup/setup-english.md).

@Basilisk3 The goal of the changes in #5000 was to keep the DPS the same, but make the tech 3 Seraphim sub less obstructed by torpedo defenses. Hence, we increased the number of torpedo's but we reduced their damage. In #5834 @lL1l1 noticed that we actually made a mistake where the salvo duration was higher than the reload time, which is of course strange 😃 .

A bit of background on the topic. I'm fine either way. @Tagada14 / @MrRowey can you raise this with the balance team, see if the tech 3 subs are indeed too strong now that torpedo's can retarget?

**Comment by Basilisk3** [comment]
Another wall of text, I apologize, but I think it is important that my you fully understand why such a drastic change is necessary.

@zqh9843 Judging from your message, I assume you used both formations in close range of each other.
- Being in range of Tech 2 subs is the worst case scenario for the Yathsou, yet it still managed to win
- With micro/usage of its range, the Yathsou is not beatable by any of the Tech 2 subs and easily wins against equivalent formations. In simpler terms, other Tech 2 subs have no chance against the Yathsou. This holds true for all naval units I tested.

Looking at the stats, the Yathsou is essentially a Barracuda but with 20 % more DPS per mass, powerful anti-air and 20 more range(!). And the Barracuda is by no means considered a bad unit, I have heard it being too strong more times than the other way round.

> However imo those doomstack scenarios aren't very likely since both sides are investing like at least 30000 mass into the sub

30000 mass = 10 subs. Tech 3 Navy is expensive, if the game progresses to this stage, it is a common sight.

> a team with air player can easily invest that much into torp bomber productions

That is an unfair comparison, as the other team would have an air player too. Either way, torpedo bombers are underperforming against the Yathsou because of its incredible torpedo defense.

> @Basilisk3 The goal of the changes in #5000 was to keep the DPS the same, but make the tech 3 Seraphim sub less obstructed by torpedo defenses. Hence, we increased the number of torpedo's but we reduced their damage. In #5834 @lL1l1 noticed that we actually made a mistake where the salvo duration was higher than the reload time, which is of course strange 😃 .

This was what I had assumed as well. Its overall DPS was still increased, though, even compared to its pre May 30, 2023 state. And with the increased amount of torpedoes, it can overwhelm torpedo defenses more easily.

Just to make sure, I am not trying to blame anyone here, obviously. This is the result of various issues being fixed, and the unit just needs to be toned down now.

**Comment by Garanas** [comment]
Giving the kiting capabilities (20 more range is about 2.5 factories) the seraphim tech 3 submarine can be quite strong, I see your point there. Let's wait a few days for the balance team to discuss it and then we'll take things from there 😃

**Comment by MrRowey** [comment]
I've asked the team to discuss this now :)
